dxr/hgext/GPL-LICENSE.txt

```diff
@@ -0,0 +1,278 @@
+        GNU GENERAL PUBLIC LICENSE
```

Comment:
I think hgext is the exception in that it is under the GPL; the rest falls under MIT.